app/src/main/kotlin/com/wire/android/ui/home/newconversation/common/SelectParticipantsButtonsRow.kt

```diff
@@ -74,14 +85,49 @@ fun SelectParticipantsButtonsRow(
     onMoreButtonClick: (() -> Unit)? = null,
     onMainButtonClick: () -> Unit,
 ) {
-    SelectParticipantsButtonsRow(count, mainButtonText, false, elevation, modifier, onMoreButtonClick, onMainButtonClick)
+    SelectParticipantsButtonsRow(
+        count = count,
+        mainButtonText = mainButtonText,
+        shouldAllowNoSelectionContinue = false,
+        elevation = elevation,
+        modifier = modifier,
+        onMoreButtonClick = onMoreButtonClick,
+        onMainButtonClick = onMainButtonClick
+    )
+}
+
+@Composable
+fun SendContentButton(
+    mainButtonText: String,
+    count: Int,
```

Comment:
could we give it somre mroe meaningfull name ? 
also is "main" prefix realy needed here ?